src/analyses/dependence_graph.h

```diff
@@ -111,6 +112,16 @@ class dep_graph_domaint:public ai_domain_baset
     data_deps.clear();
   }
 
+  const depst& get_control_deps()
+  {
+    return control_deps;
+  }
+
+  const depst& get_data_deps()
+  {
+    return data_deps;
+  }
+
```

Comment:
In order to extract the dependency information from the graph in order to create our input dependency, we need access to the data (ie control_deps and data_deps). These are getters for those structures.